darts/models/forecasting/nbeats.py

```diff
@@ -30,48 +31,72 @@ class _TrendGenerator(nn.Module):
 
     def __init__(self,
                  expansion_coefficient_dim,
-                 output_dim):
+                 target_length,
+                 nr_params):
         super(_TrendGenerator, self).__init__()
-        self.T = nn.Parameter(
-            torch.stack([(torch.arange(output_dim) / output_dim)**i for i in range(expansion_coefficient_dim)], 1),
-            False)
+
+        # coefs is of size (expansion_coefficient_dim, target_length)
+        coefs = torch.stack([(torch.arange(target_length) / target_length)**i for i in range(expansion_coefficient_dim)], 
```

Comment:
Nitpicking, but `coefs` might not be the best name here because the paper refers to the values that are multiplied with this matrix as 'expansion coefficients' already, so this might be confusing. Maybe we could call it `basis`... what do you think?